pickee_api/views.py

```diff
@@ -16,29 +16,9 @@ def home(request):
 
 
 def recommendation(request):
-    data = json.dumps({
-        "user": get_user_data(request),
```

Comment:
Please modify the `get_user_data` helper method to indicate it's meant for internal use `__get_user_data`

Reply:
Done